SU-SOE/stanford_soe_helper
# 7.x 2.x soe 2074

# READY FOR REVIEW

# Summary
- Changed all requested icons from .png to .svg files
- Added all necessary and related styles to newly added .svg files
- Updated vh calculations where necessary to accommodate new, tighter brand bar

# Needed By (Date)
- Before code push

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the newly added icons are rendering (I can point to specific places as needed)
3. Ensure that the vh calculations are working by testing that the images touch the bottom of the viewport where required (again, can point to places as needed)
4. This branch is currently on /dev

# Affected Projects or Products
- SOE
- stanford_soe_helper

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2074

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
